### PR TITLE
Add Brian Bockelman's CHEP presentation.

### PR DESCRIPTION
**Pull request by bbockelm**

### CODE REVIEW

**Comment by henryiii** [comment]
Could you add a location, please? Thanks!

File: _data/people/bbockelm.yml [hunk 1]
```diff
@@ -47,3 +47,10 @@ presentations:
     meeting: HSF & ATLAS Joint Event Delivery Workshop
     url: https://indico.cern.ch/event/849155/sessions/323697/attachments/1917032/3169811/HSF-EventDelivery-WG-Introduction.pdf
     focus-area: doma
+  - title: "Third-party transfers in WLCG using HTTP"
+    date: 2019-11-05
+    meeting: 24th International Conference on Computing in High Energy & Nuclear Physics
+    url: https://indico.cern.ch/event/773049/contributions/3474402/attachments/1938592/3213446/CHEP19-HTTP-TPC.pdf
+    meetingurl: https://indico.cern.ch/event/773049/
```
Comment (henryiii) [inline _data/people/bbockelm.yml:54]:
```suggestion
    meetingurl: https://indico.cern.ch/event/773049/
    location: Adelaide, South Australia
```

**Comment by bbockelm** [comment]
@henryiii - happy to add this, but why doesn't Travis yell at me?

**Comment by henryiii** [comment]
It’s just a warning in the logs because not all of them have locations let. It will once we have all the locations in. :)

**Pull request merged**